spec/fixtures/invalid-typechecked/tslint.json

```diff
@@ -0,0 +1,5 @@
+{
+  "rules": {
+    "no-unused-variable": true
```

Comment:
Oh, I hadn’t even considered that. That does make a lot of sense. I’ll update the PR. Thanks!